wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,258 @@
 
 #### Sample Set Selection
 
-Here you can choose whether to use the Normal, Soft, or Drum built-in sample set. You can also [enable custom overrides](/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides) (Put hitsounds files inside the map's folder first).
+Select to use the `Normal`, `Soft`, or `Drum` built-in sample sets globally. It is possible to [enable custom overrides](/wiki/custom_sample_overrides) to use hit sound files inside the beatmap's folder first for custom hit sounds.
 
 #### Sample Set Volume
 
-Here you can set the volume of the sample set. This is important, because while you want to hear the hit sounds (an important aspect of osu! and osu!mania), you do not want the hit-sounds to drown out the actual song.
+Set the volume of the sample set globally.
 
 #### Test Sample Set
 
-Click on the buttons to test out how the sample sounds. These sounds are additive, Whistle is a combination of Normal and Whistle.
+Clicking on those buttons to test out how the sample set sounds. Four buttons are provided, namely: `Normal`, `Whistle`, `Finish`, and `Clap`. These sounds are additive; `Whistle` is a combination of `Normal` and `Whistle`.
 
-Colours
--------
+### Misc. Toggles
 
-![Colours section](img/SS_Colours.jpg "Colours section")
+The `Samples match playback rate (for fully-hitsounded maps)` checkbox will adjust the hit sounds' pitch accordingly to match the speed modifiers (both in the editor and in-game). Unchecking this will allow the hit sounds to play at its own preset pitch.
 
-These sections are used mainly to assign combo colours. Combo colours are an important part of the beatmap's aesthetic value, because there will be a lot of circles and sliders in the beatmap. Clicking a combo colour opens up your OS's colour picker (At picture, uploader used Windows OS). Choose colours that compliment the background but the colours must not camouflage into the background. Up to eight combo colours can be used, although most maps uses four. Clicking the "Remove Combo Colour" will remove the highest numbered Combo colour (in other word, reverse order 8-7-6-5-4-3-2).
+## Colours
 
-The other part of this window is assigning the playfield background colour, but this is usually a moot point because beatmaps can't be ranked without a background image which usually overrides this setting unless you are planning to use a storyboard which the background may be visible.
+![Screenshot of the colours tab](img/SS_Colours.jpg "Colours tab")
 
-Storyboarding
--------------
+Configure the combo colours and playfield's background colour.
 
-![Design section](img/SS_Storyboarding.jpg "Design section")
+### Hitcircle/Slider Combos
```

Comment:
Hit Circle?

Reply:
that's how it's spelt 
![image](https://user-images.githubusercontent.com/14264992/43368258-272bff92-9320-11e8-9310-9f850bcccf36.png)